extension/parquet/parquet_extension.cpp

```diff
@@ -853,6 +868,17 @@ static vector<unique_ptr<Expression>> ParquetWriteSelect(CopyToSelectInput &inpu
 			cast_expr->SetAlias(name);
 			result.push_back(std::move(cast_expr));
 			any_change = true;
+		} else if (input.copy_to_type == CopyToType::COPY_TO_FILE && type.id() == LogicalTypeId::VARIANT) {
+			vector<unique_ptr<Expression>> arguments;
+			arguments.push_back(std::move(expr));
+
+			auto transform_func = VariantColumnWriter::GetTransformFunction();
+			transform_func.bind(context, transform_func, arguments);
+
+			auto func_expr = make_uniq<BoundFunctionExpression>(transform_func.return_type, transform_func,
```

Comment:
This is missing `func_expr->SetAlias(name)`, to produce the same name as the input expression